src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -3755,28 +3755,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         Private Sub CheckInterfaceUnificationAndVariance(diagnostics As DiagnosticBag)
             Dim interfaces = Me.InterfacesAndTheirBaseInterfacesNoUseSiteDiagnostics
 
-            Dim declaringSyntax = Me.GetDeclaringSyntaxNode(Of VisualBasicSyntaxNode)()
```

Comment:
@AlekseyTs 
I'm not sure this change is correct (although it caused no visible test failure).
I ran into assertion failing inside `GetDeclaringSyntaxNode`, for example when `this` is `My.MyComputer` (it has an empty set of `DeclaringSyntaxReferences`, but `IsImplicitlyDeclared` is false).

I'll stop by to discuss.